my-app/src/FetchRequest.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Child Class 5 - executes HTTP GET request using fetch API
+ * Functional Component 
+ */
+
+export const FetchRequest = (props) => {
+
+console.log(props.keyword);
+console.log(props.searchStatus);
+
+if (props.searchStatus === true) {
+
+  const wordQuery = props.keyword;
+  const apiKey = '9990ead4';
+  const url = 'http://www.omdbapi.com/?';
+  const queryParams = 's='; 
+ 
+  const endpoint = url + 'apikey=' + apiKey + '&' +  queryParams + wordQuery; 
+
+  fetch(endpoint)
+    .then(response => response.json())   
+    .then(data => {
```

Comment:
This can be a future task, but it'd be nice to make `FetchRequest` separate from React so that it's just vanilla Javascript. This could be achieved by passing the promise resolve/reject as a parameter to the function - this would also make it possible to get different behaviour if this function needed to be called in more than one place.